pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/integration/ChannelGroupIntegrationTest.kt

```diff
@@ -0,0 +1,140 @@
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@Suppress("UndocumentedPublicClass")
+class ChannelGroupIntegrationTest : BaseChatIntegrationTest() {
+    @Test
+    fun addChannels() = runTest {
+        chat.createChannel(channel01.id).await()
+        chat.createChannel(channel02.id).await()
+
+        val channelGroup = chat.getChannelGroup(randomString())
+        channelGroup.addChannels(listOf(channel01, channel02)).await()
+
+        val expectedChannelIds = setOf(channel01.id, channel02.id).toSet()
+        val getChannelsResponse = channelGroup.listChannels().await()
+        val actualChannelIds = getChannelsResponse.channels.map { it.id }.toSet()
+
+        assertEquals(expectedChannelIds, actualChannelIds)
+        chat.pubNub.deleteChannelGroup(channelGroup.id).await()
+    }
+
+    @Test
+    fun addChannelIdentifiers() = runTest {
+        chat.createChannel(channel01.id).await()
+        chat.createChannel(channel02.id).await()
+
+        val channelGroup = chat.getChannelGroup(randomString())
+        channelGroup.addChannelIdentifiers(listOf(channel01.id, channel02.id)).await()
+
+        val expectedChannelIds = setOf(channel01.id, channel02.id).toSet()
+        val getChannelsResponse = channelGroup.listChannels().await()
+        val actualChannelIds = getChannelsResponse.channels.map { it.id }.toSet()
+
+        assertEquals(expectedChannelIds, actualChannelIds)
+        chat.pubNub.deleteChannelGroup(channelGroup.id).await()
+    }
+
+    @Test
+    fun removeChannels() = runTest {
+        chat.createChannel(channel01.id).await()
+        chat.createChannel(channel02.id).await()
+
+        val channelGroup = chat.getChannelGroup(randomString())
+        channelGroup.addChannels(listOf(channel01, channel02)).await()
+        channelGroup.removeChannels(listOf(channel01, channel02)).await()
+
+        val getChannelsResponse = channelGroup.listChannels().await()
+        assertTrue { getChannelsResponse.channels.isEmpty() }
+        chat.pubNub.deleteChannelGroup(channelGroup.id).await()
+    }
+
+    @Test
+    fun removeChannelIdentifiers() = runTest {
+        chat.createChannel(channel01.id).await()
+        chat.createChannel(channel02.id).await()
+
+        val channelGroup = chat.getChannelGroup(randomString())
+        channelGroup.addChannels(listOf(channel01, channel02)).await()
+        channelGroup.removeChannelIdentifiers(listOf(channel01.id, channel02.id)).await()
+
+        val getChannelsResponse = channelGroup.listChannels().await()
+        assertTrue { getChannelsResponse.channels.isEmpty() }
+        chat.pubNub.deleteChannelGroup(channelGroup.id).await()
+    }
+
+    @Test
+    fun whoIsPresent() = runTest {
+        val channelGroup = chat.getChannelGroup(randomString())
+        channelGroup.addChannels(listOf(channel01, channel02)).await()
+        delayInMillis(1000)
+
+        val autoCloseable = channelGroup.connect { message -> }
```

Comment:
What is the role of this line?
Is this needed?

Reply:
Yes, it's needed. It simulates a scenario where someone is joining a channel. 
This is required to test `whoIsPresent()` functionality